### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by copa2**
Obvious Fix

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@copa2 Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=541)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=541) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/faq) for frequently asked questions.

File: README.adoc [hunk 1]
```diff
@@ -305,11 +305,11 @@ a modified file in the correct place. Just commit it and push the change.
 If you don't have an IDE preference we would recommend that you use
 http://www.springsource.com/developer/sts[Spring Tools Suite] or
 http://eclipse.org[Eclipse] when working with the code. We use the
-http://eclipse.org/m2e/[m2eclipe] eclipse plugin for maven support. Other IDEs and tools
+http://eclipse.org/m2e/[m2eclipse] eclipse plugin for maven support. Other IDEs and tools
```
Comment (spencergibb) [inline README.adoc]: These fixes for m2eclipse actually have their source here: https://github.com/spring-cloud/spring-cloud-build/blob/master/docs/src/main/asciidoc/building-base.adoc

**Comment by copa2** [comment]
Sorry didn't read the documentation part properly.
JCE 8 link is already fixed, but seems not be generated into the actual docuementation. 

I close this PR and will create one for the m2eclipe typo on the spring-cloud-build project.

**Comment by ryanjbaxter** [comment]
@copa2 I updated the README so the link should be fixed now, thanks for making us aware it needed to be updated!

**Pull request closed without merging**